netstate/routes/net-state-files.go

```diff
@@ -0,0 +1,104 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package routes
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+
+	"storj.io/storj/storage/boltdb"
+)
+
+type NetStateRoutes struct {
+	DB *boltdb.Client
+}
+
+type Message struct {
+	Value string `json:"value"`
+}
+
+func NewNetStateRoutes(db *boltdb.Client) *NetStateRoutes {
+	return &NetStateRoutes{DB: db}
+}
+
+func (n *NetStateRoutes) Put(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	givenPath := ps.ByName("path")
+
+	var msg Message
+	decoder := json.NewDecoder(r.Body)
+	err := decoder.Decode(&msg)
+	if err != nil {
+		http.Error(w, "bad request: err decoding response", http.StatusBadRequest)
+		log.Printf("err decoding response: %v", err)
```

Comment:
I think it might be useful to add a zap logger to the `NetStateRoutes` struct and initialize when you call `NewNetStateRoutes`